index.bs

```diff
@@ -4437,7 +4457,10 @@ When they are, the effect will be as you would expect.
 
 <pre class="grammar" id="prod-AsyncIterable">
     AsyncIterable :
+        "async" "iterable" "&lt;" TypeWithExtendedAttributes "&gt;" ";"
+        "async" "iterable" "&lt;" TypeWithExtendedAttributes "&gt;" "(" ArgumentList ")" ";"
         "async" "iterable" "&lt;" TypeWithExtendedAttributes "," TypeWithExtendedAttributes "&gt;" ";"
+        "async" "iterable" "&lt;" TypeWithExtendedAttributes "," TypeWithExtendedAttributes "&gt;" "(" ArgumentList ")" ";"
```

Comment:
You'll need to split this up to limit lookahead.

Reply:
Can you (or someone else watching) help me figure out how to do this? I haven't tried to learn all the LL(1) constraints and how to work around them.

Reply:
Here's one way, making use of existing sync iterable infrastructure like [OptionalType](https://heycam.github.io/webidl/#prod-OptionalType)

```
AsyncIterable : 
  async iterable < TypeWithExtendedAttributes OptionalType > OptionalArgumentList ;
OptionalArgumentList
  ( ArgumentList )
  ε
```